homeassistant/components/vesync/__init__.py

```diff
@@ -80,20 +84,27 @@ async def async_setup_entry(hass, config_entry):
     hass.data[DOMAIN][VS_MANAGER] = manager
 
     switches = hass.data[DOMAIN][VS_SWITCHES] = []
+    fans = hass.data[DOMAIN][VS_FANS] = []
 
     hass.data[DOMAIN][VS_DISPATCHERS] = []
 
```

Comment:
We usually store these per config entry `hass.data[DOMAIN][entry.entry_id][VS_FANS]` in case the user wants to setup multiple instances.

The integration currently prevents setting up multiple instances so its not an issue here.  If there is a desire to support multiple instances in the future, it may be easier to refactor now instead of when there are many platforms.

Reply:
I am fine with refactoring it, but do you think it would make sense to do as a separate pull request?

Reply:
New PR works for me 👍